GherkinSpec.TestAdapter/TestDiscoverer.cs

```diff
@@ -16,6 +17,9 @@ namespace GherkinSpec.TestAdapter
     [FileExtension(".exe")]
     public class TestDiscoverer : ITestDiscoverer
     {
+        private readonly HashSet<string> discoveredFullyQualifiedTestNames = new HashSet<string>();
+        private readonly Regex uniqueTestCaseSuffixRegex = new Regex(" \\((\\d+)\\)$");
```

Comment:
uniqueTestCaseSuffixRegex can be static?

Reply:
Thanks, made this static on `UniqueTestCaseNameResolver`.